stubs/tornado/@python2/tornado/locks.pyi

```diff
@@ -11,35 +12,37 @@ class Condition(_TimeoutGarbageCollector):
     def notify_all(self): ...
 
 class Event:
-    def __init__(self): ...
-    def is_set(self): ...
+    def __init__(self) -> None: ...
+    def is_set(self) -> bool: ...
     def set(self): ...
     def clear(self): ...
     def wait(self, timeout: Optional[Any] = ...): ...
 
 class _ReleasingContextManager:
-    def __init__(self, obj): ...
-    def __enter__(self): ...
+    def __init__(self, obj) -> None: ...
+    def __enter__(self) -> None: ...
     def __exit__(self, exc_type, exc_val, exc_tb): ...
 
 class Semaphore(_TimeoutGarbageCollector):
-    def __init__(self, value: int = ...): ...
-    def release(self): ...
+    def __init__(self, value: int = ...) -> None: ...
+    def release(self) -> None: ...
     def acquire(self, timeout: Optional[Any] = ...): ...
-    def __enter__(self): ...
+    def __enter__(self) -> NoReturn: ...
```

Comment:
The implementation is:

```python
    def __enter__(self) -> None:
        raise RuntimeError("Use 'async with' instead of 'with' for Semaphore")
```
It might be better to delete this, so that you get an error when type-checking, not at runtime.

Reply:
Done